upload/lambdas/health_check/health_check.py

```diff
@@ -0,0 +1,109 @@
+from datetime import datetime, timedelta
+import json
+import os
+
+import boto3
+import requests
+
+from upload.common.database import _run_query
+from upload.common.logging import get_logger
+from upload.common.upload_config import UploadConfig
+
+logger = get_logger(__name__)
+
+client = boto3.client('cloudwatch')
+
+
+class HealthCheck:
+    def __init__(self):
+        self.env = os.environ['DEPLOYMENT_STAGE']
+        logger.debug(f"Running a health check for {self.env}. Results will be posted in #upload-service")
+        self.webhook = UploadConfig().slack_webhook
+
+        self.checksumming_areas_count_query = "SELECT COUNT(*) FROM checksum " \
+                                              "WHERE status='CHECKSUMMING' " \
+                                              "AND created_at > CURRENT_DATE - interval '4 weeks' " \
+                                              "AND updated_at > CURRENT_TIMESTAMP - interval '2 hours'"
+        self.validation_areas_count_query = "SELECT COUNT(*) FROM validation " \
+                                            "WHERE status='VALIDATING' " \
+                                            "AND created_at > CURRENT_DATE - interval '4 weeks' " \
+                                            "AND updated_at > CURRENT_TIMESTAMP - interval '2 hours'"
+
+        self.undeleted_areas_count_query = "SELECT COUNT(*) FROM upload_area " \
+                                           "WHERE created_at > CURRENT_DATE - interval '4 weeks' " \
+                                           "AND status != 'DELETED'"
+
+    def health_check(self):
```

Comment:
I suspect that this Lambda might grow quickly with lots of different checks.  To keep it organized I would suggest you keep each check in a separate function that returns a string, and keep an array of the check functions that the primary entry point function iterates through, then joins all the results into the Slack post.  You can do this as a refactor later.

Reply:
I added the other checks in response to parth's comment before I saw this so I added the refactor here as well. I can roll back those changes and submit them in a separate pr if you would prefer.